ui/src/submissions/authors/components/AuthorForm.jsx

```diff
@@ -69,6 +69,12 @@ class AuthorForm extends Component {
               placeholder="For non-Latin names e.g. 麦迪娜 or Эдгар Бугаев"
               component={TextField}
             />
+            <Field
+              name="former_name"
+              label="Alternate/former name"
```

Comment:
@StellaCh can we go for only `Former name` here?

Reply:
We'll change it to "Alternate name" only.